plugins/UM3NetworkPrinting/resources/qml/ClusterControlItem.qml

```diff
@@ -389,6 +424,14 @@ Component
                                         radius: 0.5 * width
                                         color: UM.Theme.getColor("viewport_background")
                                     }
+                                    contentItem: Label
+                                    {
+                                        text: contextButton.text
+                                        color: UM.Theme.getColor("monitor_text_inactive")
+                                        font.pixelSize: 25
```

Comment:
Is screen scaling needed here?

Reply:
Yes, this should use pointSize instead of pixelSize